packages/verified-fetch/src/verified-fetch.ts

```diff
@@ -9,6 +9,7 @@ import { peerIdFromString } from '@libp2p/peer-id'
 import { Key } from 'interface-datastore'
 import { exporter } from 'ipfs-unixfs-exporter'
 import toBrowserReadableStream from 'it-to-browser-readablestream'
+import { LRUCache } from 'lru-cache'
```

Comment:
nit: we have `src/utils/tlru.ts` we could use instead of pulling in another dep

Reply:
`src/utils/tlru.ts` is based on `hashlru` which doesn't have the equivalent of `lru-cache`'s `dispose` option which we need to clean up sessions before they are evicted from the cache.

TBH `lru-cache` is 10x more downloaded on npm than `hashlru` and it does everything `src/utils/tlru.ts` does, maybe we should swap that out for `lru-cache`?

Reply:
good callout, lets do it